src/apps/btc/btc_sign.c

```diff
@@ -305,8 +307,13 @@ app_btc_sign_error_t app_btc_sign_input(
     if (_state != STATE_INPUTS_PASS1 && _state != STATE_INPUTS_PASS2) {
         return _error(APP_BTC_SIGN_ERR_STATE);
     }
-    if (request->sequence != 0xffffffff) {
-        return _error(APP_BTC_SIGN_ERR_INVALID_INPUT);
+    if (request->sequence < 0xffffffff - 1) {
+#if defined(APP_BTC)
```

Comment:
I think this has changed in https://github.com/digitalbitbox/bitbox02-firmware/pull/251